conda/gateways/disk/create.py

```diff
@@ -128,16 +128,15 @@ def create_python_entry_point(target_full_path, python_full_path, module, func):
         'import_name': import_name,
     }
     if python_full_path is not None:
-        shebang = u'#!%s\n' % python_full_path
+        from ...core.portability import generate_shebang_for_entry_point
```

Comment:
It would be wonderful to have this at the top of the file.

Reply:
I didn't put it there because `conda.core` is "off-limits" within `conda.gateways`, according to the README :(

Reply:
Le sigh.